gtwrap/matlab_wrapper/mixins.py

```diff
@@ -1,19 +1,22 @@
 """Mixins for reducing the amount of boilerplate in the main wrapper class."""
 
+from typing import Any, Tuple, Union
+
 import gtwrap.interface_parser as parser
 import gtwrap.template_instantiator as instantiator
 
 
 class CheckMixin:
     """Mixin to provide various checks."""
     # Data types that are primitive types
-    not_ptr_type = ['int', 'double', 'bool', 'char', 'unsigned char', 'size_t']
+    not_ptr_type: Tuple = ('int', 'double', 'bool', 'char', 'unsigned char',
```

Comment:
Why use `tuple`? I think these are better suited as arrays.

Reply:
Tuples are immutable. Since we are only checking for membership, this was a good way to ensure constness.